lib/node_modules/@stdlib/string/next-grapheme-cluster-break/README.md

```diff
@@ -0,0 +1,174 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2020 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# nextGraphemeClusterBreak
+
+> Return the next extended grapheme cluster break in a string after a specified position.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var nextGraphemeClusterBreak = require( '@stdlib/string/next-grapheme-cluster-break' );
+```
+
+#### nextGraphemeClusterBreak( string, position )
+
+Returns the next extended grapheme cluster break in a string after a specified position.
+
+```javascript
+var out = nextGraphemeClusterBreak( 'last man standing', 4 );
+// returns 5
+```
+
+</section>
+
+<!-- /.usage -->
+
+<!-- Package usage notes. Make sure to keep an empty line after the `section` element and another before the `/section` close. -->
+
+<section class="notes">
+
+## Notes
+
+-   If `string` is an empty string, the function returns `0` irrespective of `position`.
+
+</section>
+
+<!-- /.notes -->
+
+<!-- Package usage examples. -->
+
+<section class="examples">
+
+## Examples
+
+<!-- eslint no-undef: "error" -->
+
+```javascript
+var nextGraphemeClusterBreak = require( '@stdlib/string/next-grapheme-cluster-break' );
+
+console.log( nextGraphemeClusterBreak( 'last man standing', 4 ) );
+// => 5
+
+console.log( nextGraphemeClusterBreak( 'presidential election', 8 ) );
+// => 9
+
+console.log( nextGraphemeClusterBreak( 'अनुच्छेद', 1 ) );
+// => 3
+
+console.log( nextGraphemeClusterBreak( '🌷', 0 ) );
+// => 2
+```
+
+</section>
+
+<!-- /.examples -->
+
+<!-- Section for describing a command-line interface. -->
+
+* * *
+
+<section class="cli">
+
+## CLI
+
+<!-- CLI usage documentation. -->
+
+<section class="usage">
+
+### Usage
+
+```text
+Usage: next-grapheme-cluster-break [options] [<string>] --pos=<index>
```

Comment:
If `pos` is allowed to be optional, then `pos` no longer needs to be a required option.